src/Kart/LeafletTangram/scene/visualisering/polygon.js

```diff
@@ -4,7 +4,7 @@ import sysconfig from '../../../../config'
 function drawAll(drawArgs) {
   const { kode, barn, farge, opplystKode, visBarn, visEtiketter } = drawArgs
   const layer = {
-    data: { source: kode, layer: kode },
+    data: { source: kode, layer: 'vector4326' },
```

Comment:
Needs to be given a more sensible name

Reply:
Can be fixed by adding -l $layerName here:
https://github.com/Artsdatabanken/grunnkart-dataflyt/blob/master/bin/export/tippecanoe.sh#L6

Reply:
https://github.com/Artsdatabanken/grunnkart-dataflyt/issues/68